src/ui-test/basicServerTest.ts

```diff
@@ -51,8 +49,11 @@ export function wildflyE2EBasicTest() {
                 expect(serversNames).to.include.members([serverName]);
                 const server = await serverProvider.getServer(serverName);
                 await server.start();
+                await VSBrowser.instance.driver.wait( async () => { return await serverHasState(server, ServerState.Started);}, 3000 );
```

Comment:
`driver` is already initialized as a shorthand for `VSBrowser.instance.driver`, why not use it?

Reply:
true, true....